src/Apps/__stories__/ConversationApp.story.tsx

```diff
@@ -0,0 +1,22 @@
+import { MockRouter } from "DevTools/MockRouter"
+import React from "react"
+import { storiesOf } from "storybook/storiesOf"
+import { conversationRoutes } from "../Conversation/routes"
+
+storiesOf("Apps/Conversation", module)
+  .add("Default", () => {
+    return (
+      <MockRouter
+        routes={conversationRoutes}
+        initialRoute="/user/conversations"
+      />
+    )
+  })
+  .add("Detail", () => {
```

Comment:
This sort of feels like a hard storybook entry to maintain, especially since we don't have consistent fixtures that live forever on staging, etc.

It's probably sufficient to just provide the storybooks entry for the Inbox as a whole (as you've done above).